tasks.py

```diff
@@ -0,0 +1,48 @@
+#! /bin/env python3
```

Comment:
If we're going to include the shebang for the purposes of just running `./tasks.py` in the terminal:
* It should be `#! /usr/bin/env python`, not just `/bin/env`
* The file should be executable

Otherwise, we should require that it is executed via `python3 tasks.py` per usual.

Lmk what you want to do.